HTAN.model.csv

```diff
@@ -650,7 +648,7 @@ Lymph Nodes Tested,The number of lymph nodes tested to determine whether lymph n
 Lymphatic Invasion Present,"A yes/no indicator to ask if small or thin-walled vessel invasion is present, indicating lymphatic involvement","Yes, No, unknown, Not Reported",,,FALSE,Diagnosis,,https://cdebrowser.nci.nih.gov/cdebrowserClient/cdeBrowser.html#/search?publicId=64171&version=3.0,
 Metastasis at Diagnosis,The text term used to describe the extent of metastatic disease present at diagnosis.,"Distant Metastasis, Metastasis NOS, No Metastasis, Regional Metastasis, unknown, Not Reported",,,FALSE,Diagnosis,,https://cdebrowser.nci.nih.gov/cdebrowserClient/cdeBrowser.html#/search?publicId=6133614&version=1.0,
 Metastasis at Diagnosis Site,Text term to identify an anatomic site in which metastatic disease involvement is found.,"Abdomen, Adrenal Gland, Ascites, Bone, Bone Marrow, Brain, Cerebrospinal Fluid, Central Nervous System, Colon, Distant Nodes, Distant Organ, Groin, Kidney, Liver,  Lung, Lymph Node, Axillary, Lymph Node, Inguinal, Lymph Node NOS, Mediastinum,  Omentum, Ovary, Pelvis, Peritoneal Cavity,  Peritoneum,  Pleura, Scalp, Skin, Small Intestine, Soft Tissue, Spinal Cord, unknown, Not Reported",,,FALSE,Diagnosis,,https://cdebrowser.nci.nih.gov/cdebrowserClient/cdeBrowser.html#/search?publicId=3029815&version=1.0,
-Method of Diagnosis,Text term used to describe the method used to confirm the patients malignant diagnosis.,"Autopsy,Biopsy,Blood Draw,Bone Marrow Aspirate,Core Biopsy,Cytology,Cystoscopy,Debulking,Diagnostic Imaging,Dilation and Curettage Procedure,Enucleation,Excisional Biopsy,Fine Needle Aspiration,Imaging,Incisional Biopsy,Laparoscopy,Laparotomy,Other,Pap Smear,Physical Exam,Pathologic Review,Surgical Resection,Thoracentesis,Ultrasound Guided Biopsy,unknown,Not Reported",,,FALSE,Diagnosis,,https://cdebrowser.nci.nih.gov/cdebrowserClient/cdeBrowser.html#/search?publicId=6161031&version=1.0,
+Method of Diagnosis,Text term used to describe the method used to confirm the patients malignant diagnosis.,"Autopsy,Biopsy,Blood Draw,Bone Marrow Aspirate,Core Biopsy,Cytology,Cystoscopy,Debulking,Diagnostic Imaging,Dilation and Curettage Procedure,Enucleation,Excisional Biopsy,Fine Needle Aspiration,Incisional Biopsy,Laparoscopy,Laparotomy,Other,Pap Smear,Physical Exam,Pathologic Review,Surgical Resection,Thoracentesis,Ultrasound Guided Biopsy,unknown,Not Reported",,,FALSE,Diagnosis,,https://cdebrowser.nci.nih.gov/cdebrowserClient/cdeBrowser.html#/search?publicId=6161031&version=1.0,

```

Comment:
Presume we are using `Imaging` from valid values as it is causing a namespace conflict?
Can we replace with `Diagnostic Imaging` rather than remove in its entirety.
Will this cause any submitted manifests to fail.